notes/0.13.10/always-honor-api-url.markdown

```diff
@@ -0,0 +1,10 @@
+[@jkinkead]: https://github.com/jkinkead
+[2262]: https://github.com/sbt/sbt/pull/2262
+
+### Fixes with compatibility implications
+
+### Improvements
+
+- The setting `apiURL` used to be ignored if `autoAPIMappings` was `false`. It will now be inserted into the built POM file regardless of the value of `autoAPIMappings`.  [#2262] by [@jkinkead]
```

Comment:
It looks like the link to the PR does not show up as a link. In the other notes files it is done this way:

```markdown
[#2262][2262]
```